sonaranalyzer-dotnet/src/SonarAnalyzer.Common/SymbolicExecution/Constraints/CollectionCapacityConstraint.cs

```diff
@@ -0,0 +1,38 @@
+﻿/*

```

Comment:
Alright forget about my other comment on file being deleted, I guess Git didn't detected the move correctly.